relay-general/src/protocol/debugmeta.rs

```diff
@@ -377,13 +377,11 @@ pub struct NativeDebugImage {
     /// Starting memory address of the image (required).
     ///
     /// Memory address, at which the image is mounted in the virtual address space of the process. Should be a string in hex representation prefixed with `"0x"`.
-    #[metastructure(required = "true")]
```

Comment:
Does this work with the sentry processor as is? I think we may expect non-null values at some places in `native/processing.py` or `native/symbolicator.py`.

Reply:
Looked into this, and it will work. There are two places in AppleCrashReport rendering where we would throw None `ValueError`s.